### PR TITLE
Update Sif layer mediaType for ORAS

### LINKED ISSUE #3278: confusing messaging when removing a remote
## Version of Singularity:

d95ecd3c41915a988f615e33d2e5fbeeabdb4522

### Expected behavior

`singularity remote remove xxx` either removes `xxx` or tells me where it's looking for it and why it's not found.

### Actual behavior

`singularity remote remove xxx` listed in the output of `singularity remote list` does not exist.

### Steps to reproduce behavior

```
$ builddir/singularity remote list
NAME           URI              SYS
[SylabsCloud]  cloud.sylabs.io  YES

$ builddir/singularity remote remove SylabsCloud
FATAL:   SylabsCloud is not a remote

$ builddir/singularity remote remove '[SylabsCloud]'
FATAL:   [SylabsCloud] is not a remote
```

Note how it tells me that `SylabsCloud` is not a remote, even if it's present in the output of `list`.

The reason why it's saying it's not a remote is because `SylabsCloud` is found in the system's `remote.yaml` file, not in my own one. If I add `-g` to the command, it does find it.

The error message should at least say that the remote is not found in the user's configuration (vs the global one).

### LINKED ISSUE #4437: Clarify and correct OCI Artifact Type for SIF
Hey folks! I want to pass forward some important discussion that needs your attention - on the OCI board there is a somewhat incorrect representation of the content type string for Singularity (SIF), specifically it has a spelling mistake and the incorrect format (tar). The original discussion is here: https://github.com/opencontainers/image-spec/issues/791#issuecomment-531062035 and I think it would be really important for someone that official represents Singularity (at Sylabs) to step in and offer opinion / wisdom. Thanks!

### PR DESCRIPTION
**Pull request by dtrudg**
## Description of the Pull Request (PR):

Use the format discussed in the reference on #4437 going forward. Allow
our existing 'prototype' mediaType to be handles for images already
pushed into ORAS.

### This fixes or addresses the following GitHub issues:

 - Fixes #4437 

#### Before submitting a PR, make sure you have done the following:

- Read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: e2e/pull/pull.go [hunk 1]
```diff
@@ -371,7 +390,9 @@ func checkPullResult(t *testing.T, tt testStruct) {
 // this is a version of the oras push functionality that does not check that given the
 // file is a valid SIF, this allows us to push arbitrary objects to the local registry
 // to test the pull validation
-func orasPushNoCheck(file, ref string) error {
+// We can also set the layer diaType - so we can push images with older media types
```
Comment (ikaneshiro) [inline e2e/pull/pull.go]:
```suggestion
// We can also set the layer MediaType - so we can push images with older media types
```

**Review by ikaneshiro — APPROVED** [review]
Lgtm, just a typo in a comment

**Comment by dtrudg** [comment]
Corrected for the conversation in #4437 around the `.sif` instead of `+sif` at the end of the mediaType

**Comment by dtrudg** [comment]
Okay - let's do this. Think it's pretty uncontroversial to match the opencontainers example :-)

**Pull request merged**